catalog/resource_metastore_test.go

```diff
@@ -36,7 +36,9 @@ func TestCreateMetastore(t *testing.T) {
 				Method:   "PATCH",
 				Resource: "/api/2.0/unity-catalog/metastores/abc",
 				ExpectedRequest: map[string]interface{}{
-					"owner": "administrators",
+					"owner":                 "administrators",
+					"delta_sharing_enabled": true,
+					"delta_sharing_recipient_token_lifetime_in_seconds": 0,
```

Comment:
That's the longest field name I've seen so far at Databricks

Reply:
🤣